### ISSUE TITLE
Is this still active?

### ISSUE THREAD

**Issue by jamesthesnake** [JuliaIO/NIfTI.jl#28]
Haven't see development for a while?

**Comment by Tokazama** [comment]
Yes and I'm hoping to get my branch ready for a pull request soon. I intended to finish it over the summer but my PhD program decided to give me a few surprises that kept me very busy.

In fact, I've spent the majority of today implementing the [Brain Imaging and Data Structure Format](https://bids.neuroimaging.io/) in another package as it turns out there are a variety of file extensions that NIfTI uses. I'll lay out a more complete plan later today in issue #16 .
